TreeMaker/python/makeTreeFromMiniAOD_cff.py

```diff
@@ -642,7 +646,29 @@ def makeTreeFromMiniAOD(
                                          'JetsProperties:neutralHadronMultiplicity(Jets_neutralHadronMultiplicity)',
                                          'JetsProperties:photonMultiplicity(Jets_photonMultiplicity)',
                                          'JetsProperties:flavor(Jets_flavor)'])
-
+    if IncludeBEff:
+    	process.bTaggingEff        = cms.EDAnalyzer('BTaggingEffAnalyzer',
+        	JetsTag            = cms.InputTag('HTJets'),
+        	DiscriminatorTag   = cms.string('pfCombinedInclusiveSecondaryVertexV2BJetTags'),
+        	DiscriminatorValue = cms.double(0.890),
+        	PtNBins            = cms.int32(100),
+        	PtMin              = cms.double(0.),
+        	PtMax              = cms.double(1000.),
+        	EtaNBins           = cms.int32(60),
+        	EtaMin             = cms.double(-3.),
+        	EtaMax             = cms.double(3.)
+    	)
+   	process.Baseline += process.bTaggingEff
+    if AddBtagSF:
+	process.BTagScale=cms.EDProducer('BTagScale',
+		JetsTag            = cms.InputTag('HTJets'),
+		BTagEffInput   =cms.string('./data/test_run_RA2AnalysisTree.root'),
```

Comment:
For now that file is a test file that is also part of the pull request. The file-naming convention will follow the treemaker convention for the signal scan: Just need to input Mg, mLSP
-Rishi
> On Oct 5, 2015, at 5:26 PM, Kevin Pedro <notifications@github.com> wrote:
> 
> In TreeMaker/python/makeTreeFromMiniAOD_cff.py <https://github.com/TreeMaker/TreeMaker/pull/68#discussion_r41157247>:
> 
> > +    	process.bTaggingEff        = cms.EDAnalyzer('BTaggingEffAnalyzer',
> > +        	JetsTag            = cms.InputTag('HTJets'),
> > +        	DiscriminatorTag   = cms.string('pfCombinedInclusiveSecondaryVertexV2BJetTags'),
> > +        	DiscriminatorValue = cms.double(0.890),
> > +        	PtNBins            = cms.int32(100),
> > +        	PtMin              = cms.double(0.),
> > +        	PtMax              = cms.double(1000.),
> > +        	EtaNBins           = cms.int32(60),
> > +        	EtaMin             = cms.double(-3.),
> > +        	EtaMax             = cms.double(3.)
> > +    	)
> > +   	process.Baseline += process.bTaggingEff
> > +    if AddBtagSF:
> > +	process.BTagScale=cms.EDProducer('BTagScale',
> > +		JetsTag            = cms.InputTag('HTJets'),
> > +		BTagEffInput   =cms.string('./data/test_run_RA2AnalysisTree.root'),
> Do we need to track this ROOT file "test_run_RA2AnalysisTree.root"? If so, could it be given a more descriptive name?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/TreeMaker/TreeMaker/pull/68/files#r41157247>.
>